frontend/prebuild/package-lock.json

```diff
@@ -13604,8 +13604,7 @@
     "uglify-to-browserify": {
       "version": "1.0.2",
       "resolved": "https://registry.npmjs.org/uglify-to-browserify/-/uglify-to-browserify-1.0.2.tgz",
-      "integrity": "sha1-bgkk1r2mta/jSeOabWMoUKD4grc=",
-      "optional": true
+      "integrity": "sha1-bgkk1r2mta/jSeOabWMoUKD4grc="
```

Comment:
Do you know what this change is for? Never seen it do that before but if it's something you did then ok.

Reply:
nope, this was not intentional..